modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/service/test/CalendarBookingLocalServiceTest.java

```diff
@@ -506,6 +506,41 @@ public void testAddCalendarBookingResourseRequestedOverlappingEnd()
 		Assert.assertTrue(secondChildCalendarBooking.isDenied());
 	}
 
+	@Test
+	public void testAddCalendarBookingVideoDescription() throws Exception {
```

Comment:
I think we could rename to `testAddCalendarBookingWithVideoDescription` because based on the patterns I found here when there is a specific quirk in the test the "With" word should come first, make sense to you?

Thanks!